docs/develop/07_dApp/02_session.md

```diff
@@ -25,25 +30,9 @@ The dapp introduces itself to the extension with its name, encryption key URI, a
 A copy of the challenge should be stored on the server side.
 For example:
 
-```ts
-import { FullDidDetails } from '@kiltprotocol/did'
-
-const did = 'did:kilt:example'
-const dAppName = 'Your dApp Name'
-
-const fullDid = await FullDidDetails.fromChainInfo(did)
-
-const dAppEncryptionKeyUri = fullDid.assembleKeyUri(fullDid.encryptionKey.id)
-
-// generate and store challenge on the server side for the next step
-const challenge = await fetch('/challenge')
-
-const session = await window.kilt.sporran.startSession(
-  dAppName,
-  dAppEncryptionKeyUri,
-  challenge
-)
-```
+<TsJsSnippet>
+  {DappIntroduction}
+</TsJsSnippet>
```

Comment:
The imports are missing from the code example. Would be helpful to include them. :innocent:

Reply:
The thing is that before the function body there are few declared variables that make the whole TS file compile. If we show the whole file, including the imports, these would be shown and it is not good. We don't have component that can reliably exclude some lines from the output, at the moment. We can either show the whole file, or a function body given its name and its start and end delimiter.